generator-honeycomb/test/app.js

```diff
@@ -197,20 +214,27 @@ describe('generator-honeycomb', () => {
         before((done) => {
           if (type === 'prompts') {
             prompts.templateEngine = 'react';
-            helpers.run(appPath).withPrompts(prompts).on('end', done);
+            helpers
+              .run(appPath)
+              .withPrompts(prompts)
+              .on('end', done);
           } else if (type === 'options') {
             options.template = 'react';
-            helpers.run(appPath).withOptions(options).on('end', done);
+            helpers
+              .run(appPath)
+              .withOptions(options)
+              .on('end', done);
           }
         });
 
         it('creates expected files', () => {
           assert.file([
-            'src/client/components/Example.jsx',
-            'src/client/client.jsx',
-            'src/server/views/index/index.jsx',
+            'src/client/components/Example.js',
+            'src/client/client.js',
```

Comment:
Please add this file (`src/client/client.js`) at the top `it` block because it's the same for both engines